src/components/UI/Tabs/Tabs.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import React, {Fragment} from 'react';
+
+import {Divider} from '../Divider/Divider';
+import {Tab} from '../Tab/Tab';
+
+export const Tabs = ({tabs}) => {
+  return tabs.map((tab, index) => {
+    return (
+      <Fragment key={index}>
```

Comment:
assign `index` as the key is not the best practice.

from: https://reactjs.org/docs/lists-and-keys.html : 
`We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny’s article for an [in-depth explanation on the negative impacts of using an index as a key](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/). If you choose not to assign an explicit key to list items then React will default to using indexes as keys.`

Reply:
Fixed